src/impl/domain.ts

```diff
@@ -234,6 +234,18 @@ export function getId(): string | undefined {
   return activeDomain && activeDomain[ID_KEY];
 }
 
+/**
+ * Gets the short unique domain id created for the current context / scope.
+ *
+ * @returns {(string | undefined)}
+ */
+export function getShortId(): string | undefined {
+  const activeDomain = getActiveDomain();
+  const id = activeDomain && activeDomain[ID_KEY];
+
```

Comment:
No need to duplicate the code when you could do:
```suggestion
  const id = getId();
```